lib/get-groups.js

```diff
@@ -1,3 +1,5 @@
-module.exports = function getGroups (str) {
-  return str.split(/[:\s+]/)
-}
+"use strict";
+
+module.exports = function getGroups(str) {
+  return str.split(/[:\s+]/);
+};
```

Comment:
fari enough, lets put the output in a folder called 'dist' rather than src then, to avoid any confusion. then we can ignore that folder in the package.json standard config and in the gitignore